### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by YuyaoHu**

### CODE REVIEW

**Review by Cloudwu1 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+Updated native SDK versions:
+
+* iOS from 2.3.0 to [2.4.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.4.0)
+
+### App Center Auth
+
+#### iOS
+
+* **[Fix]** Redirect URIs are now hidden in logs.
```
Comment (jaeklim) [inline CHANGELOG.md]: This version introduces Auth. We don't need to mention Feature/Fix here. Please refer to RN CHANGELOG. We are introducing Data in RN so you should get an idea for this CHANGELOG there.

Comment (Cloudwu1) [reply CHANGELOG.md]: But Unity uses DotNet nuget packages, not RN's.

Comment (guperrot) [reply CHANGELOG.md]: What Jae meant here, it's the first release of Auth for Unity. The first release does not "fix" anything.

The data was just an example.

Here is the better example you need:

https://github.com/microsoft/appcenter-sdk-android/blob/develop/CHANGELOG.md#appcenterauth

Comment (Cloudwu1) [reply CHANGELOG.md]: @guperrot Thanks for your explanation.  We push in the new commit. Could you help to review this again?

**Review by Cloudwu1 — APPROVED** [review]

**Pull request merged**